src/v2/BaseLoginRouter.js

```diff
@@ -105,6 +106,11 @@ export default Router.extend({
     if (lastResponse) {
       sessionStorageHelper.setStateHandle(idxResponse?.context?.stateHandle);
     }
+    // Ignore Device Probe calls that can occur on first page loads
+    // that mimic moving forward in remediation flow
```

Comment:
Let's generalize this comment, since we are in BaseLoginRouter.. and a jira ticket number. 

`Login Flows that mimic step up (moving forward in login pipeline) via internal api calls, needs to clear stored stateHandles. This way the flow can maintain the latest state handle.`